src/components/FullArticlePage.jsx

```diff
@@ -0,0 +1,43 @@
+import { useParams } from 'react-router-dom'
+import { useState, useEffect } from "react"
+
+import {getArticleById} from '../utils/api-utils'
+
+const FullArticlePage = () => {
+const [article, setArticle] = useState([])
```

Comment:
Could we have some loading state in here to let the user know when the article is being fetched?